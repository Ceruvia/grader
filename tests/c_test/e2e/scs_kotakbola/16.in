103 609 754
H 36
M 12
B 19
M 17
H 35
E
