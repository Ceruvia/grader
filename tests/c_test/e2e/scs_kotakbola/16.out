71 29 19
Ya
